imbi/templates/openapi.yaml

```diff
@@ -2389,8 +2389,6 @@ paths:
                     type: string
                     enum:
                       - create-project
-                      - create-project-dependency
-                      - remove-project-dependency
```

Comment:
I think removing these was accidental, probably need to pull in the latest openapi changes.

Reply:
there are a few other changes associated with it as well that are reverted in this commit

Reply:
😞 but it's an easy fix